beaconChain/pow/fake.ts

```diff
@@ -0,0 +1,46 @@
+import {Deposit, DepositData, DepositInput} from "../interfaces/blocks";
+
+type int = number;
+type hash32 = Uint8Array;
+
+interface DummyChainStart {
```

Comment:
I might use the names `ChainStart`, `log`, or some combination of those, etc. for names of interfaces/functions in this file.
Based on: https://github.com/ethereum/eth2.0-specs/blob/master/specs/core/0_beacon-chain.md#chainstart-log

Reply:
Not sure what you mean here.